modules/datadict/jsx/dataDictIndex.js

```diff
@@ -159,8 +232,9 @@ class DataDictIndex extends Component {
     return (
         <FilterableDataTable
            name="datadict"
-           data={this.state.data.Data}
+           data={this.state.data}
            fields={fields}
+           loading={this.state.isLoading}
```

Comment:
When I was working on making streaming work, I developed a `progress bar` that calculates the percent completion of the stream and displays it on the front end. Perhaps I can add this functionality on afterward so that the user knows how close all the data is to being loaded.

Reply:
You can add it for other uses, but I don't think that would work here because there's no way to know how much data there is until you get to the end. I initially had it start with a 64 bit int to keep track of the progress, but that requires going through the data twice to first calculate the number of rows and then to actually send it. (Doing it based on byte size would have a similar problem.)